src/peergos/server/util/PeergosNetworkUtils.java

```diff
@@ -364,6 +365,86 @@ public static void shareFileWithDifferentSigner(NetworkAccess sharerNode,
         Assert.assertEquals(friendChildren.size(), 1);
     }
 
+    public static void shareAndUnshare(NetworkAccess sharerNode) throws Exception {
```

Comment:
we already have share and unshare tests, maybe a clearer name is sharedwithPermutations?